### PR TITLE
bionemo version is set statically in one file, used throughout

### PR DESCRIPTION
**Pull request by malcolmgreaves**
Changes how the `version` attribute for all bionemo sub-packages is assigned. Previously, the version string was computed from a the state of `git tag` in the repository. Specifically, the tag for the current commit -- or the last such tag -- would be used as the version. This caused prbolems whenever the tag was not of the right format. Additionally, it made it hard to pin down exactly where the version is coming from.

A simplier mechanism is in place. Now, the version always comes from the `VERSION` file at the root level. To make installation work as expected, each `sub-package` has a symlink to the root-level `VERSION` file.

An additional change is made in this PR: all LICENSE files also now symlink to the root-level LICENSE. This better ensures that all sub-packages are using the same verbatim license text.

### CODE REVIEW

**Review by tshimko-nv — APPROVED** [review]

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-2.0.0
+2.0
```
Comment (dorotat-nv) [inline VERSION:1]: super, version changed

**Review by dorotat-nv — APPROVED** [review]

**Comment by dorotat-nv** [comment]
great cleanup

File: VERSION [hunk 2]
```diff
@@ -1 +1 @@
-2.0.0
+24.10
```
[the remark this answers is not in the document]
Comment (dorotat-nv) [reply VERSION]: @pstjohn , the version has been updated to `2.0`

**Review by polinabinder1 — APPROVED** [review]

**Comment by malcolmgreaves** [comment]
Successful CI run: https://github.com/NVIDIA/bionemo-framework/actions/runs/11556730836/job/32165106058

**Pull request merged**